src/views/design/ScriptDetail/index.tsx

```diff
@@ -300,6 +301,18 @@ const ScriptDetail = withStyles(styles)(
                                     disableUnderline: true,
                                 }}
                             />
+                            <TextInput
+                                id="script-securityGroup"
```

Comment:
use - as word seperator instead of camelcase to be aligned with other 'id's. so script-security-group